lib/liquid_voting_web/schema/schema.ex

```diff
@@ -19,9 +19,9 @@ defmodule LiquidVotingWeb.Schema.Schema do
       resolve(&Resolvers.Voting.participants/3)
     end
 
-    @desc "Get a participant by its id"
+    @desc "Get a participant by its uuid"
```

Comment:
Should this be `id`? (I've noticed both `uuid` and `id` being used (also in tests))

Reply:
I created a new issue for this, yesterday.

Reply:
Where in tests (other than organization_uuid)? I've just been through all the tests again, and can't see any.

Reply:
OK, found it. There was one in voting_results_test.ex.